src/com/nutrons/nu17/subsystems/Drivetrain.java

```diff
@@ -1,104 +1,115 @@
 package com.nutrons.nu17.subsystems;
```

Comment:
Check the formatting on this entire class.  Comments should end on the line before the class starts.  Indentation should be consistent.  Fix this stuff in this PR